src/Fixer/LanguageConstruct/ClassKeywordRemoveFixer.php

```diff
@@ -159,7 +159,7 @@ private function replaceClassKeyword(Tokens $tokens, $classIndex)
         $classBeginIndex = $classEndIndex;
         while ($tokens[--$classBeginIndex]->isGivenKind(array(T_NS_SEPARATOR, T_STRING)));
         ++$classBeginIndex;
-        $classString = $tokens->generatePartialCode($classBeginIndex, $classEndIndex);
+        $classString = ltrim($tokens->generatePartialCode($classBeginIndex, $classEndIndex), '\\');
```

Comment:
Do you mean like this?
```php
$classStringBeginIndex = $tokens[$classBeginIndex]->isGivenKind([T_NS_SEPARATOR]) ? $classBeginIndex + 1 : $classBeginIndex;
$classString = $tokens->generatePartialCode($classStringBeginIndex, $classEndIndex);
```

This would require a new index-variable, because the `$classBeginIndex` is still required to call clearRange and insertAt later in the method.